store/sqlstore/channel_store.go

```diff
@@ -3490,6 +3490,38 @@ func (s SqlChannelStore) CreateSidebarCategory(userId, teamId string, newCategor
 	}, nil
 }
 
+func (s SqlChannelStore) completePopulatingCategoryChannels(category *model.SidebarCategoryWithChannels) (*model.SidebarCategoryWithChannels, *model.AppError) {
+	if category.Type == model.SidebarCategoryCustom || category.Type == model.SidebarCategoryFavorites {
+		return category, nil
+	}
+
+	var sql string
+	var args []interface{}
```

Comment:
Any reason to pre-declare them like this ?

Reply:
no. residue from old code. will cleanup